src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
-.catalog {
-  display: flex;
-  flex-wrap: wrap;
+@import "./utils/variables";
 
-  max-width: 944px;
+.catalog {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 48px;
+  justify-items: center;
   margin: 0 auto;
+  max-width: $container-main;
+}
 
-  &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
-    border: 1px solid #ccc;
-    border-radius: 5px;
-    margin-bottom: 48px;
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: 1fr 1fr;
   }
+}
```

Comment:
Use scss nesting and move all media inside catalog class so that you can avoid repeating of catalog class